docs/root/intro/arch_overview/http/upgrades.rst

```diff
@@ -14,7 +14,10 @@ pass through the default HTTP filter chain. To avoid the use of HTTP-only filter
 one can set up custom
 :ref:`filters <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.UpgradeConfig.filters>`
 for the given upgrade type, up to and including only using the router filter to send the HTTP
-data upstream.
+data upstream. Note that the :ref:`Buffer filter <envoy_v3_api_msg_extensions.filters.http.buffer.v3.Buffer>` breaks
```

Comment:
s/the Buffer filter breaks upgrades/buffering is generally not compatible with upgrades/

I think you could arguably have an H2 upgrade that worked with the buffer filter but it would be unusual, not the norm.

Reply:
Updated -- thanks!